src/command/dev/next/mod.rs

```diff
@@ -150,10 +166,18 @@ impl Dev {
             .watch_for_changes(write_file_impl, composition_messages)
             .await;
 
+        warnln!(
+            "Do not run this command in production! It is intended for local development only."
+        );
+
+        infoln!("your supergraph is running! head to {router_address} to query your supergraph");
+
         while let Some(router_log) = run_router.router_logs().next().await {
             match router_log {
                 Ok(router_log) => {
-                    eprintln!("{}", router_log);
+                    if !router_log.to_string().is_empty() {
+                        eprintln!("{}", router_log);
+                    }
```

Comment:
This is mostly for tidiness, but the router seems to print a lot of blank lines, which doesn't help us out, so I've filtered them out